llvm/test/tools/yaml2obj/GOFF/GOFF-oneSymb.yaml

```diff
@@ -0,0 +1,40 @@
+# RUN: yaml2obj %s | od -v -An -tx1 | FileCheck --ignore-case %s
+
+# Verify that GOFF Header is correct.
```

Comment:
I'd focus on the parts of the header that might vary, rather than testing the whole thing.